blocks/feed-newsroom/feed-newsroom.css

```diff
@@ -90,7 +94,7 @@ main .feed-newsroom-wrapper .load-more-container {
 }
 
 @media (min-width: 62rem) {
-main .section.feed-newsroom-container.narrow > .section-container {
+main .section.feed-newsroom-container.narrower > .section-container {
```

Comment:
If this is the "default" style for the `feed-newsroom-container`, could we remove the `narrower` style from this selector and remove it from the section as well?

Reply:
Seems like this narrower behaviour is automatically coming from the fragment which is there in this section, hence I cant remove that 

<img width="846" alt="image" src="https://github.com/hlxsites/sunstar/assets/37831850/378a643e-7572-4967-a7cd-92b7b97ef7f8">